RDMMonitor.js

```diff
@@ -581,8 +706,35 @@ async function SendDMAlert(device)
     return;
 }
 
+function SendRebootAlert(device)
+{
+    if(!config.sendRebootAlerts)
+    {
+        return;
+    }
+    else
+    {
+        for(var i = 0; i < config.userAlerts.length; i++)
+        {
+            let user = bot.users.get(config.userAlerts[i]);
+            if(!user)
+            {
+                console.error(GetTimestamp()+"Cannot find a user to DM with ID: "+config.userAlerts[i]);
+            }
+            else
+            {
+                user.send(GetTimestamp()+" Device: "+device+" was sent the reboot command").catch(error => {
+                    console.error(GetTimestamp()+"Failed to send a DM to user: "+user.id);
+                    return;
+                });
+            }
+        }
+        return;
+    }

```

Comment:
```suggestion
```
- Remove closing bracket to unnecessary `else` statement
- Remove unnecessary return statement from end of method/function